insecure/net/network_ingress_test.go

```diff
@@ -0,0 +1,96 @@
+package net
```

Comment:
Can you clarify why you think it's better to put this in a test package?

Reply:
Renamed 2 test helper files in insecure/net to differentiate from regular files:
`insecure/net/mock_attacker_test_helper.go`
`insecure/net/network_test_helper.go`
So as a convention all `*_test_helper.go` files are test helper files

Reply:
Suggestion: you may try to move the helpers into the `internal` package (read [here](https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface)), i.e., `insecure/net/internal`. In this way we gonna have three packages: `net`, `net_test`, and `internal` subpackage.

Reply:
Thanks for the suggestion. I tried this but I'm getting an import cycle error because the helper functions use `corruptnet.Network` (the parent package) and the parent package imports the helper package.

`imports github.com/onflow/flow-go/insecure/corruptnet/internal: import cycle not allowed in test`